ogre2/src/terrain/Terra/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ file( GLOB_RECURSE TERRA_SOURCES
 
 add_library(${PROJECT_NAME} STATIC ${TERRA_SOURCES})
 
-if(IGN_ADD_fPIC_TO_LIBRARIES AND NOT _ign_add_library_INTERFACE)
+if(IGN_ADD_fPIC_TO_LIBRARIES AND NOT _gz_add_library_INTERFACE)
```

Comment:
Humm using CMake variables that were meant to be private 😕 

Good that this is migrated here though 🙂 Upstream already changed in 

* https://github.com/gazebosim/gz-cmake/pull/245

Reply:
Yepp